services/api/src/admin_email_notification.py

```diff
@@ -0,0 +1,181 @@
+import logging
+import common.pgquery as pgquery
+import sqlalchemy
+import os
+
+
+def get_notification_data(user_email):
+    query = (
+        "SELECT to_jsonb(row) "
+        "FROM ("
+        "SELECT u.email, u.first_name, u.last_name, e.email_type "
+        "FROM public.user_email_notification "
+        "JOIN public.users AS u ON u.user_id = user_email_notification.user_id "
+        "JOIN public.email_type AS e ON e.email_type_id = user_email_notification.email_type_id "
```

Comment:
I initially ran into API errors when visiting the Notifications page because there was already an existing database and the `email_type` table didn't exist. Upon clearing the volume and re-deploying the CV Manager, this error no longer occurred.

We may want to consider the case where users are upgrading to a new version and provide some sort of migration script to ease the transition.

Reply:
Added a migration script under /scripts/update_scripts